nservicebus/upgrades/7to8/index.md

```diff
@@ -249,4 +246,69 @@ if (extensions.TryGet(out DiscardIfNotReceivedBefore constraint))
 {
     timeToBeReceived = constraint.MaxTime;
 }
-```
\ No newline at end of file
+```
+
+## Dependency on System.Memory package for .NET Framework
+
+Memory allocations for incoming and outgoing messages bodies are reduced by using the low allocation memory types via the System.Memory namespace. These type are available on .NET Framework via the **System.Memory** package. The NServiceBus build that targets .NET Framework has this dependency added.
+
+### Pipeline context types changes
+
+Throughput the pipeline all contexts have been updated to use `ReadOnlyMemory<byte>` (e.g. context for behaviors, stage connectors, and message mutators) and affects the following types:
```

Comment:
```suggestion
Throughout the pipeline, all context types (e.g. context for behaviors, stage connectors, and message mutators) have been updated to use `ReadOnlyMemory<byte>` instead of `byte[]`. These are:
```